sdk/network/Azure.ResourceManager.Network/tests/Helpers/NetworkServiceClientTestBase.cs

```diff
@@ -599,6 +685,11 @@ protected ApplicationGatewayContainer GetApplicationGatewayContainer(string reso
             return GetResourceGroup(resourceGroupName).GetApplicationGateways();
         }
 
+        protected ApplicationGatewayContainer GetApplicationGatewayContainer(Resources.ResourceGroup resourceGroup)
```

Comment:
Do we really need this method? I think the caller can just replace with `resourceGroup.GetApplicationGateways()`.